_rules/focusable-no-keyboard-trap-80af7b.md

```diff
@@ -56,6 +54,8 @@ _There are no major accessibility support issues known for this rule._
 
 ## Background
 
+This rule only applies to HTML and SVG. Thus, it is a partial check for WCAG 2.0 success criterion 2.1.2, which applies to all content.
```

Comment:
I'm wondering whether we should completely remove this note instead.
It is essentially true for every single rule we have that it applies only to HTML and sometimes SVG and therefore is a partial check for WCAG… But we're not putting the note in every rule. Why would this rule specifically need it?

(also if kept we should probably refer to WCAG 2.1 and link to the SC)

Reply:
Agree that there is nothing to make this rule special. I'm removing it.